cardigan/stories/components/BetaMessage.js

```diff
@@ -10,6 +10,6 @@ stories.add(
     <BetaMessage message="We are working to make this item available online in April 2019." />
   ),
   {
-    info: Readme,
+    readme: { sidebar: Readme },
```

Comment:
I wonder if there's a way to do this for all stories, and break if there isn't one?

Reply:
Nice idea. I'll stick it on a list.